config/locales/views/saving_goals/pt-BR.yml

```
@@ -0,0 +1,93 @@
+      target: "Meta: %{amount}"
+      deadline: "Prazo: %{date}"
+      on_track: "No prazo"
+      behind: "Atrasado"
+      contributions: "Contribuições"
+      actions: "Ações"
+      edit: "Editar Meta"
+      delete: "Excluir Meta"
+      target_date: "Data alvo: %{date}"
+      current_balance: "Saldo Atual"
+      of_target: "da meta de %{amount}"
+      on_track: "No Prazo"
```

YAMLlint (1.38.0)

[error] duplication of key "on_track" in mapping

(key-duplicates)